Makefile

```diff
@@ -17,7 +17,7 @@ format:
 
 .PHONY: test_e2e
 test_e2e:
-	pytest -vv ${PYTEST_FLAGS} tests/e2e
+	pytest -vv -n 10 ${PYTEST_FLAGS} tests/e2e
```

Comment:
run tests 10-parallel

Reply:
hmm, maybe this introduced flakes? we could try 5-parallel, or even revert this change.

Reply:
5 runs in a row without failures (restarted ma manually), all seems to be OK :)